platform/security/filter/security-filter-csrf/src/main/java/org/codice/ddf/security/filter/csrf/CsrfFilter.java

```diff
@@ -48,6 +48,7 @@
   private static final String APACHE_USER_AGENT = "Apache-CXF/.*";
   private static final String JAVA_CLIENT_USER_AGENT = "Google-HTTP-Java-Client/.*";
   private static final String JAVA_USER_AGENT = "Java/.*";
+  private static final String JETTY_USER_AGENT = "Jetty/.*";
```

Comment:
❓ Is there a good reason not to move these into the blueprint as a second configurable list?

Reply:
There isn't

Reply:
I'll move it into blueprint